src/pages/contributing/product-development-lifecycle/index.mdx

```diff
@@ -0,0 +1,145 @@
+
+#### PDLC applied to contribution
+Carbon has applied the PDLC framework to how we expect new assets like components, features, and patterns to be contributed to Carbon. We have broken the contribution process up into the same three phases, each having their own set of success criteria.
+
+- **Discovery phase**: The purpose of the discovery phase is to address any major value, usability, feasibility and viability risks ahead of delivering product-quality code, in order to arrive at a successful product faster and at a lower cost. 
+- **Delivery phase**: The delivery phase emphasizes the disciplined execution of determining how we will realize user value and maximizing the efficiency of building it. This includes the functional and non-functional requirements, and the user experience required for customers to adopt product features within a production setting. 
+- **Launch and scale phase**: Launch and scale of course, involves having clearly defined success criteria or learning objectives that are monitored after launch. In this phase an asset will become stable and achieve Carbon’s “definition of done.”
+
+_For IBMers only_: You can read more about the PDLC on the [IBM Winning Products](https://w3.ibm.com/w3publisher/winning-products/how-we-work/product-development-lifecycle) website.
+
+<Row>
+<Column colLg={12}>
+
+![PDLC diagram](images/PDLC-01.png)
+
+</Column>
+</Row>
+
+## Discovery 
+
+The discovery phase is where research, exploration, and validation happen. This is when innovations to the system are proposed and reviewed.
+
+### Discovery criteria 
+
+For an enhancement or net new asset to warrant resourcing a discovery phase, we need to determine the requirements for an asset to be considered as an innovation. Proposals need to show that the component or pattern would be useful to many teams and unique to the system. 
+
+#### Key considerations:
+
+- Does it replicate anything in the system already, or is there truly a gap?
+- If the proposal does replicate an existing asset, is there evidence to show that the proposed solution is better?
+- Is there already an existing issue or proposal in [GitHub](https://github.com/carbon-design-system/carbon/issues) to address the gap?
+- Is there evidence that the new asset or enhancement would be useful for many teams or services?
+- What is the ratio of feasibility to impact to help prioritize (consult developers and accessibility SMEs)?
+
+#### Discovery status
+
+All assets in the discovery phase start as drafts. As the asset progresses in its completeness and validation it can graduate from `draft` to `preview candidate`. This means its ready to be reviewed and seriously considered for inclusion as a maintained asset in the system. Carbon is not the gatekeeper of the discovery phase; this phase is fully curated by the community and innovators. 
```

Comment:
```suggestion
All assets in the discovery phase start as drafts. As the asset progresses in its completeness and validation, it can graduate from `draft` to `preview candidate`. This means it's ready to be reviewed and seriously considered for inclusion as a maintained asset in the system. Carbon is not the gatekeeper of the discovery phase; this phase is fully curated by the community and innovators. 
```